### DISCUSSION TITLE
Panel with fixed size

### DISCUSSION THREAD

**Discussion by ronildo** [bvaughn/react-resizable-panels#326] [Q&A] [answered]
Is there a way to have two panels so that the left side doesn't change if the window size changes

For example on Shadcn website, if you remove the container class from the website, it expands to the whole screen.
From the two images you can see that the left sidebar changed because the window size changed.

![Screenshot 2024-03-23 at 5 40 15 PM](https://github.com/bvaughn/react-resizable-panels/assets/622159/9295a69b-30d9-4aaf-bdab-17312be9e89b)

![Screenshot 2024-03-23 at 5 40 36 PM](https://github.com/bvaughn/react-resizable-panels/assets/622159/3b4e594e-24f2-4976-9957-2a1f64e7a41d)

**Comment by bvaughn** [comment]
The only way to lock a panel's size is set min and max size to the same value.

The primary purpose of this library is _resizable_ panels though, so I'm not really sure why you'd want to do that. :)

**Reply by bvaughn** [reply, marked as answer]
FYI more recent releases of this library support two methods of locking a panel's size:

- Rendering a div: https://react-resizable-panels.vercel.app/examples/fixed-size-panels
- Rendering it as disabled: https://react-resizable-panels.vercel.app/examples/disabled-panels
